src/components/Categories/LoanSpotlight.vue

```diff
@@ -0,0 +1,270 @@
+<template>
+	<div>
+		<h2 class="md:tw-hidden tw-pb-2">
+			Today's loan spotlight
```

Comment:
I like leaving this text up while it loads, good choice! the loading state looks great